contacts.permissions.yml

```diff
@@ -3,6 +3,10 @@ administer contacts:
   description: 'Configure the contacts dashboard and related settings.'
   restrict access: TRUE
 
+manage contacts dashboard:
+  title: 'Manage contacts dashboard'
+  description: 'Add, remove and rearrange dashboard tab content.'
```

Comment:
Could probably just be 'Configure dashboard tabs and tab content', as we'll also be able to configure them?